tlog-witness.md

```diff
@@ -0,0 +1,226 @@
+# The Witness Protocol
+
+This document describes a synchronous HTTPS-based protocol to obtain
```

Comment:
This doc refers to HTTPS, which I think would be a recommended deployment strategy. But I think that naked HTTP would still be secure?

Reply:
Yep, and and on-path attacker could do the same with HTTPS too.